1.9/administration/metrics/index.md

```diff
@@ -3,4 +3,142 @@ post_title: Metrics
+| usage.Total       | Available capacity in bytes. |
+| usage.Used        | Capacity used in bytes.      |
+| usage.InodesFree  | Available Inodes in btyes.   |
+| usage.InodesTotal | Available Inodes in bytes.   |
+| usage.InodesUsed  | Inodes used in bytes.        |
+
+###  Node
+
+#### Metrics
+   
+| Metric            | Description                  |
+|-------------------|------------------------------|
+| cpu.cores         |    Percentage of cores used.     |
+| cpu.idle         |     Percentage of CPUs idle.         |
+| cpu.system         |    Percentage of system used.   |
+| cpu.total         |   Percentage of CPUs used.  |
+| cpu.user         |   Percentage of CPU used by the user.   |
+| cpu.wait         |   Percentage idle while waiting for an operation to complete.    |
+| load.1min         |     Load average for the past minute.       |
+| load.5min         |   Load average for the past 15 minutes 5 minutes.        |
+| load.15min         |    Load average for the past 15 minutes.        |
+| memory.buffers         |   Number of memory buffers.     |
+| memory.cached         |   Amount of cached memory.   |
+| memory.free         |    Amount of free memory in bytes.   |
+| memory.total         |   Total memory in bytes.   |
+| processes         |  Number of processes that are running.          |
+| swap.free         |  Amount of free swap space.   |
+| swap.total         |  Total swap space.    |
+| swap.used         |    Amount of swap space used.    |
+| uptime          |   The system reliability and load average.    |
+   
+#### Filesystem
+   
+| Metric            | Description                  |
+|-------------------|------------------------------|
+| filesystem.{{.Name}}.capacity.free    | Amount of available capacity in bytes. |
+| filesystem.{{.Name}}.capacity.total    | Total capacity in bytes. |
+| filesystem.{{.Name}}.capacity.used    |  Capacity used in bytes. |
+| filesystem.{{.Name}}.inodes.free    | Amount of available inodes in bytes. |
+| filesystem.{{.Name}}.inodes.total    | Total inodes in bytes. |
+| filesystem.{{.Name}}.inodes.used    | Inodes used in bytes.  |
+      
+#### NetworkInterface
+   
+| Metric            | Description                  |
+|-------------------|------------------------------|
+| network.{{.Name}}.in.bytes    | Number of bytes downloaded. |
+| network.{{.Name}}.in.dropped    | Number of downloaded bytes dropped. |
+| network.{{.Name}}.in.errors    | Number of downloaded bytes in error. |
+| network.{{.Name}}.in.packets    | Number of packets downloaded. |
+| network.{{.Name}}.out.bytes    | Number of bytes uploaded. |
+| network.{{.Name}}.out.dropped    | Number of uploaded bytes dropped. |
+| network.{{.Name}}.out.errors    | Number of uploaded bytes in error.  |
+| network.{{.Name}}.out.packets    | Number of packets uploaded. |
+   
+### Container metrics
+
+Container metrics define the structure of the response expected from Mesos when referring to container and executor metrics.
```

Comment:
I think this might be better summarized by something like the following:

> Per-container resource utilization metrics

In my opinion, using "structure" and "response" in this section are more related to the implementation than what I would expect to read as a user looking at these docs.